solutions/usaco-687.mdx

```diff
@@ -83,4 +83,63 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+```py
+"""
+We're going to make a dictionary to store the milk production
+of each cow.
+Next, we're going to make a list of the dictionary
+and sort that list by milk production.
+Finally, we're going to iterate over that list
+to find who produces the second least amount of milk.
+"""
```

Comment:
not needed

Reply:
I disagree, when I was in bronze, I had a pretty hard time understanding what the code did. Having a roadmap of the solution that is more reader-friendly would help out people who are just starting in bronze in my opinion.

Reply:
the editorial exists